tests/php/bootstrap.php

```diff
@@ -31,6 +31,17 @@
 	define( 'JETPACK_WOOCOMMERCE_INSTALL_DIR', dirname( __FILE__ ) . '/../../../woocommerce' );
 }
 
+if ( "1" != getenv( 'JETPACK_TEST_WP_SUPER_CACHE' ) ) {
+	echo "To run Jetpack WP Super Cache tests, prefix phpunit with JETPACK_TEST_WP_SUPER_CACHE=1" . PHP_EOL;
+} else {
+	/**
+	 * "Mocking" function so that it exists and Jetpack_Sync_Actions will load Jetpack_Sync_Module_WP_Super_Cache
+	 */
+	function wp_cache_is_enabled() {
```

Comment:
Lets do a if function exists here in case some one runs it with the plugin installed.